src/Store/VimStoreConnector.re

```diff
@@ -173,7 +173,7 @@ let start =
       let fileType =
         switch (meta.filePath) {
         | Some(v) =>
-          Some(Exthost.LanguageInfo.getLanguageFromFilePath(languageInfo, v))
+          Some(Exthost.LanguageInfo.getLanguageFromFilePath(languageInfo, v)) // TODO: I should be fromBuffer..but I use metadata right now.
```

Comment:
I think this is reasonable, for now - this code is hit when the filename changes